src/partials/_products.hbs

```diff
@@ -2,43 +2,54 @@
 	<h2>Products</h2>
 
 	<!----------Toy Products---------->
```

Comment:
Cool little thing. If this is a comment for coders reading the source HBS file, you can use a handlebars comment instead `{{!-- Comments --}}`. Those will be visible in the HBS file, but not in the HTML file. Not a must, just an FYI thing.